airflow/configuration.py

```diff
@@ -227,6 +227,33 @@ class AirflowConfigParser(ConfigParser):
         ('database', 'sql_alchemy_connect_args'): ('core', 'sql_alchemy_connect_args', '2.3.0'),
         ('database', 'load_default_connections'): ('core', 'load_default_connections', '2.3.0'),
         ('database', 'max_db_retries'): ('core', 'max_db_retries', '2.3.0'),
+        **{
+            ('kubernetes', x): ('kubernetes_executor', x, '2.4.0')
```

Comment:
This should not be `2.4.0`, given we are already at `2.4.1`, right?

Reply:
Yup thanks